interface/forms/phq9/new.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * phq-9 form using forms api     view.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once(__DIR__ . "/../../globals.php");
```

Comment:
remove above line

Reply:
hi @ruthkonyn , looks like Brady's suggest here and a few more below still need to be addressed